css/component_styles/snackbar.css

```diff
@@ -0,0 +1,30 @@
+.snackbar {
+    background-color: var(--snackbar-color);
+    border-radius: 5px;
+    display: flex;
```

Comment:
Suggestion: If you don't have a flex utility class in your CSS, You can add that and that avoids reusing here.
If you already have, please ignore this

Reply:
Everything else looks good to merge.

Reply:
ok i will do it 👍